packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -416,6 +416,12 @@ export class GraphQLTransform {
         description: 'Your GraphQL API ID.',
         exportName: Fn.join(':', [Aws.STACK_NAME, 'GraphQLApiKey']),
       });
+
+      new CfnOutput(rootStack, 'GraphQLAPIKeyExpirationOutput', {
```

Comment:
nit: I missed the part where this new output is needed/used in the implementation.